common/CHANGELOG.md

```diff
@@ -2,11 +2,11 @@
 All notable changes to the common module will be documented in this file.
 
 #### 1.x Releases (API v1)
-- `1.1.x` Releases - [1.1.0](#110)
+- `1.1.x` Releases - [1.1.1](#111)
```

Comment:
Did you do a 1.1.0 release? If so, this line should read:

- `1.0.x` Releases - [1.1.0](#110) | [1.1.1](#111)

Reply:
Thanks